jenkins/scripts/integration_test.sh

```diff
@@ -68,7 +68,7 @@ if [[ "${TESTS_FOR}" == "feature_tests"* ]] || [[ "${TESTS_FOR}" == "e2e_tests"*
 fi
 echo "Running in region: ${OS_REGION_NAME}"
 
-if [[ "${TESTS_FOR}" == "feature_tests"* || "${TESTS_FOR}" == "e2e_tests"* && "${GINKGO_FOCUS}" != "integration" ]]; then
+if [[ "${TESTS_FOR}" == "feature_tests"* || "${TESTS_FOR}" == "e2e_tests"* && ( "${GINKGO_FOCUS}" != "integration" && "${GINKGO_FOCUS}" != "basic" ) ]]; then
```

Comment:
Do we have feature_tests* anymore ?

Reply:
True We have to clean that but I guess we can do it in a separate PR it is not related to adding the basic integration test